lib/pages/portal/service/website_provider.dart

```diff
@@ -220,4 +248,26 @@ class WebsiteProvider with ChangeNotifier {
       return false;
     }
   }
+
+  void updateVisits(Website website) async {
```

Comment:
Add dartdoc comment explaining what this does, and move it next to the method that initializes the number of visits, since they're linked.